setup/quickstart/halyard-gke-public/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: single
-title:  "Public Spinnaker on GKE"
+title:  "Try out public Spinnaker on GKE"
```

Comment:
[minor and possibly to be addressed in a separate issue] Do we feel people are likely to know what "public Spinnaker" means here?